lib/much-rails/call-method.rb

```diff
@@ -0,0 +1,30 @@
+require "much-plugin"
+
+# MuchRails::CallMethod is a mix-in to implement the `call`
+# class/instance method pattern used in service objects.
```

Comment:
```suggestion
# class/instance method pattern.
```
Just end it there for me please. Don't wanna confuse with "what's a service object?" questions.